src/settings/RefreshTokenRotation.js

```diff
@@ -88,7 +85,7 @@ const RefreshTokenRotation = ({ stripes }) => {
             <FormattedMessage id="ui-developer.rtr.forceRefresh" />
           </Button>
 
-          <Form
+          {/* <Form
```

Comment:
this form currently doesn't configure anything, due to that being pulled from STCOR-888, so let's hide it.